asnets/experiments/actprop_ipc23.py

```diff
@@ -63,7 +63,7 @@
 # optionally, we can have lower limit for rollouts in case we have HUGE test
 # problems
 EVAL_TIME_LIMIT_SECONDS = int(60 * 30)
-ROUND_TURN_LIMIT = 300
+ROUND_TURN_LIMIT = 1000
```

Comment:
This looks like parameter optimization, not a bugfix. Can you explain why this can be considered a bugfix?

Reply:
Because a small ROUND_TURN_LIMIT makes the planner terminate before the time limit of 30 minutes, and it makes the planner only look for plans of length less than 300. This parameter is inherited from the original ASNets code, so we consider changing this parameter as a bug fixing.